packages/app-mobile/components/screens/JoplinCloudLoginScreen.tsx

```diff
@@ -90,6 +91,7 @@ const JoplinCloudScreenComponent = (props: Props) => {
 				if (response && response.success) {
 					dispatch({ type: 'COMPLETED' });
 					clearInterval(interval);
+					void CommandService.instance().execute('synchronize');
```

Comment:
Does it definitely work on mobile too? Since this command contains reducer actions that don't exist in mobile, and it's also quite new the support for commands on mobile

Reply:
Yes, I tested it and it was working. I will double-check if any errors are happening that I didn't see.

Reply:
I couldn't find any errors. I could change the line to:
```void reg.scheduleSync(0);```

Since ultimately it is what is happening when executing the command.